src/components/editor/index.js

```diff
@@ -44,17 +44,28 @@ const StyledWrapper = styled.div`
   cursor: ${props => props.editor.mode === EDITOR_MODES.EDIT ? 'pointer' : 'crosshair'};
 `;
 
+const AVAILABLE_LAYERS = [
```

Comment:
Shouldn't this be in `default-settings`?

Reply:
I will move it to default-settings but i will rename it into
```
EDITOR_AVAILABLE_LAYERS
```